File: problemas/iniciante/2029/2029.py
def main():
    while True:
        try:
            V = float(input())
            D = float(input())

            area = 3.14 * ((D / 2) ** 2)
            altura = V / area

            print(f"ALTURA = {altura:.2f}")
            print(f"AREA = {area:.2f}")

        except EOFError:
            break

if __name__ == "__main__":
    main()

File: problemas/iniciante/2029/2029.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let entrada = input.trim().split('\n');
    
    while (entrada.length) {
        let V = parseFloat(entrada.shift());
        let D = parseFloat(entrada.shift());
        
        D /= 2;

        let area = 3.14 * D * D;
        let altura = V / area;

        console.log(`ALTURA = ${altura.toFixed(2)}`);
        console.log(`AREA = ${area.toFixed(2)}`);
    }
});
